docs/2-virtual-machines-containers/2.3-managing-infrastructure.md

```diff
@@ -48,29 +39,29 @@ In this section, you will be taking what you've learned about virtual machines t
 - Functional testing
 - Deploying artifacts
 
-# Exercise
-
-Configure a Jenkins and Artifactory server and set up a Jenkins job to deploy a build artifact to Artifactory.
+### Exercise
 
- 1. Create a clone of your Golden Image from the previous section and [install Jenkins](https://wiki.jenkins.io/display/JENKINS/Installing+Jenkins+on+Red+Hat+distributions).
- 2. In GitHub, fork the [spring-petclinic](https://github.com/spring-projects/spring-petclinic) project.
- 3. Create a Maven job in Jenkins to build your fork of spring-petclinic.
+Configure a self-hosted GitHub Action to deploy a build artifact to Artifactory.
 
-   ?> You may need to install some dependencies for the project to build successfully.
- 4. Create another clone of your Golden Image and [install Artifactory OSS](https://www.jfrog.com/confluence/display/JFROG/Installing+Artifactory). Create a Maven repository during the Artifactory setup.
- 5. Install the Artifactory Jenkins plugin.
- 6. Add a Post-Build Action to deploy the build artifacts to your Artifactory server.
- 7. Build the spring-petclinic job and confirm the artifacts are deployed to Artifactory.
+  1. Install Artifactory into this clone
+      - Make sure to select Maven as the repository option during setup.
+  2. Create another clone of your Golden Image
+  3. Install any dependencies this image needs to be able to build the spring-petclinic project
+  4. In GitHub, fork the [spring-petclinic](https://github.com/spring-projects/spring-petclinic) project.
+  5. In the GitHub portal, add a new self-hosted runner to your fork.
+  6. Create another clone of your Golden Image
+  7. Set up this clone as your self-hosted runner in the GitHub portal
+  8. Alter the maven-build GitHub Action Workflow to use your self-hosted runner
+  9. Add a `upload` step to your workflow that uploads the build output to your Artifactory server.
+  10. Commit your changes to the fork and verify that your workflow completed successfully.
+  11. Open the admin panel for your Artifactory server and verify that you see new artifacts have been uploaded.
```

Comment:
The order of these steps seem incorrect to me. Step 1 references `this clone` what clone? Maybe we should have `Create another clone of your Golden Image` should be step 1? 

Can we also still keep a link to docs on installing Artifactory https://jfrog.com/help/r/jfrog-installation-setup-documentation/installing-artifactory

Reply:
You are correct, I will fix

Reply:
pushed a change to address all of these